src/Inventory/Conf.php

```diff
@@ -935,38 +935,51 @@ public function showConfigForm()
             );
             //if action == action_status => show blocation else hide blocaction
             echo Html::scriptBlock("
-            function changestatus() {
-                if ($('#dropdown_stale_agents_action$rand').val() != 0) {
-                $('#blocaction1').show();
-                $('#blocaction2').show();
-                } else {
-                $('#blocaction1').hide();
-                $('#blocaction2').hide();
+                function changestatus() {
+                    if ($('#dropdown_stale_agents_action$rand').val() != 0 && $('#dropdown_stale_agents_action$rand').val() != 2) {
+                        $('#bloc_status_action1').show();
+                        $('#bloc_status_action2').show();
+                    } else {
+                        $('#bloc_status_action1').hide();
+                        $('#bloc_status_action2').hide();
+                    }
                 }
-            }
-            changestatus();
-
-        ");
+                changestatus();
+            ");
             echo "</td>";
             echo "</tr>";
             //blocaction with status
-            echo "<tr class='tab_bg_1'><td colspan=2></td>";
+            echo "<tr class='tab_bg_1' style='display:none' id='bloc_status_action1'><td colspan=2></td>";
             echo "<td>";
-            echo "<span id='blocaction1' style='display:none'>";
-            echo \State::createTabEntry(__('Change the status'), 0, \State::getType());
-            echo "</span>";
+            echo \State::createTabEntry(__('If the asset status is (all if empty)'), 0, \State::getType());
+            echo "</td>";
+            echo "<td width='20%'>";
+            Dropdown::showFromArray(
+                'previous_stale_agents_status',
+                State::getStateList(),
```

Comment:
Why not put the list of statutes applicable to the inventory items directly ?
Rather than excluding other statuses with ```used```.

Reply:
fixed